.github/workflows/deploy.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - main
+      - develop
+    types:
+      - completed
+  workflow_dispatch:
```

Comment:
@junan Can we remove this? 🤔

Reply:
@carryall It helps to run the workflow manually (from the Github UI, no need to push) 
![Screen Shot 2021-01-04 at 4 16 23 PM](https://user-images.githubusercontent.com/6076762/103519535-455b0f80-4ea8-11eb-8368-12f37458ad7d.png)


https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/